modules/core/src/cuda_stream.cpp

```diff
@@ -293,6 +293,7 @@ class cv::cuda::Stream::Impl
 
     Impl();
     Impl(const Ptr<GpuMat::Allocator>& allocator);
+    Impl(const int cudaFlags);
```

Comment:
Please use type safe `unsigned int` in internal implementation.

Reply:
done